tests/nested/manual/uc20-install-in-initrd/the-modeenv

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Do we need `set -e` here somewhere?

Reply:
This is the minimal modifications from core-initrd core20 branch. I did not want to add anything that could change too much of the behavior.

Reply:
And there are lots of things in this script that I dislike.

Reply:
Yeah, sorry, I did not realize it's just copied. We share a  dislike for this script so let's just make sure we kill it eventually :)

Reply:
probably good to have TODO that when we turn on this for TPM we need to verify the reseal info:

/var/lib/snapd/boot-assets/
/var/lib/snapd/device/fde/boot-chains

see test that probe those already atm

Reply:
I have added assertions for `/var/lib/snapd/boot-assets/`. And a TODO for ` /var/lib/snapd/device/fde/boot-chains`

Reply:
thank for adding those checkes